lumicks/pylake/detail/image.py

```diff
@@ -129,7 +129,40 @@ def seek_timestamp_next_line(infowave):
     return pixel_start[idx + 1]
 
 
-def reconstruct_image_sum(data, infowave, pixels_per_line, lines_per_frame=None):
+def process_infowave(infowave):
+    # Example infowave:
+    #  1 0 0 1 0 1 2 0 1 0 0 1 0 1 0 1 0 0 1 0 2 0 1 0 1 0 0 1 0 1 0 1 2 1 0 0 1
+    #              ^ <-----------------------> ^                       ^
+    #                       one pixel
+    mask = infowave != InfowaveCode.discard
+    subset = infowave[mask]
+
+    # After discard:
+    #  1 1 1 2 1 1 1 1 1 2 1 1 1 1 1 2 1 1 1
+    #        ^ <-------> ^           ^
+    #         pixel_size (i.e. data samples per pixel)
+    # This should be:
+    #   pixel_sizes = np.diff(np.flatnonzero(infowave == InfowaveCode.pixel_boundary))
+    # But for now we assume that every pixel consists of the same number of samples
+    return subset, mask
+
+
+def round_up(size, n):
+        """Round up `size` to the nearest multiple of `n`"""
+        return int(math.ceil(size / n)) * n
+
+
+def reshape_reconstructed_image(pixels, shape):
+    # shape = (<<lines_per_frame>>, pixels_per_line)
```

Comment:
I'd make this part of the function's docstring since it's the expectation for one of the arguments. Also, a general docstring would nice here to indicate the purpose of this function -- something short.